components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenMgtDAO.java

```diff
@@ -1337,7 +1389,7 @@ public void revokeToken(String tokenId, String userId) throws IdentityOAuth2Exce
 
         String accessTokenStoreTable = OAuthConstants.ACCESS_TOKEN_STORE_TABLE;
         PreparedStatement ps = null;
-        Connection connection = IdentityDatabaseUtil.getDBConnection();;
+        Connection connection = IdentityDatabaseUtil.getDBConnection();
```

Comment:
Can result in a connection leak if an exception occurs before reaching the try block. Better to use try-with-resource or at least move the line inside the catch block.

Reply:
Fix with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/472